sql/moz-fx-data-shared-prod/telemetry_derived/desktop_eng_client_v1/query.sql

```diff
@@ -0,0 +1,28 @@
+SELECT
+  au.submission_date,
+  au.client_id,
+  cfs.first_seen_date,
+  cfs.attribution_campaign,
+  cfs.attribution_content,
+  cfs.attribution_dlsource,
+  cfs.attribution_medium,
+  cfs.attribution_ua,
+  mozfun.norm.os(cd.os) AS normalized_os,
+  cd.normalized_os_version AS os_version_major,
```

Comment:
COALESCE(
   mozfun.norm.windows_version_info(os, os_version, windows_build_number),
   NULLIF(SPLIT(normalized_os_version, ".")[SAFE_OFFSET(0)], "")
 ) as normalized_os_version

Reply:
I put your formula into the query, thank you!